tests/transactional/disable_selinux.pm

```diff
@@ -0,0 +1,59 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: transactional-update
+# Summary: Enable SELinux on transactional server
+#
+# Maintainer: QA-C team <qa-c@suse.de>
+
+use base "consoletest";
```

Comment:
```suggestion
use base "consoletest";
```
and you can remove the import of `strict` and `warnings`

Reply:
or use `use Mojo::Base "consoletest";`, it auto imports the strict and warnings libs

Reply:
yes, my suggested change was actually for Mojo, not sure why I suggested no change :D